sed/calibrator/energy.py

```diff
@@ -1523,16 +1524,24 @@ def add_offsets(
                         signs.append(v["sign"])
                     except KeyError as exc:
                         raise KeyError(f"Missing sign for offset column {k} in config.") from exc
-                    preserve_mean.append(v.get("preserve_mean", False))
-                    reductions.append(v.get("reduction", None))
+                    pm = v.get("preserve_mean", False)
+                    if pm == "false":
+                        pm = False
+                    elif pm == "true":
+                        pm = True
```

Comment:
Shouldn't the yaml library handle this correctly? Off course, these should not be strings in the yaml...

Reply:
otherwhise, also add .lower() ?

Reply:
no, it does not automatically pick up bools and None